NuGet.config.orig

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are these `.orig` files intentional?

Reply:
Nope, looks like a bad merge, I'll fix it. Thanks for catching this.